presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PredicatePushDown.java

```diff
@@ -958,5 +959,11 @@ public PlanNode visitTableScan(TableScanNode node, RewriteContext<Expression> co
 
             return node;
         }
+
+        @Override
+        public PlanNode visitAssignUniqueId(AssignUniqueId node, RewriteContext<Expression> context)
+        {
+            return context.defaultRewrite(node, context.get());
```

Comment:
I added a `checkState` as currently `uniqueId` column is used only for grouping. I didn't want to filter out this symbol as it would be a dead code without any tests - currently there is no SQL which would produce such case.